fabolaji/calculator
# Added percentage feature

**What does this PR do?**

This PR adds the percentage feature to any inserted numbers.

**How can this PR be tested?**

1. Lunch the project
2. To test this feature, you could type ```5``` and the press the percentage (%) sign
3. To get the output of step 2, click the equal to sign. In our case, expected output of ```5%``` should be equal to ```0.05```
